src/components/Calculator.js

```diff
@@ -1,31 +1,85 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable react/no-access-state-in-setstate */
+/* eslint-disable react/no-unused-state */
+/* eslint-disable react/destructuring-assignment */
 /* eslint-disable react/button-has-type */
 /* eslint-disable react/prefer-stateless-function */
 import React from 'react';
+import calculate from '../logic/calculate';
 
 export default class MyCalculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.num = '';
+    this.value = 0;
+    this.state = {
+      screen: 0, total: 0, next: 0, operation: '',
+    };
+    this.calchandler = this.calchandler.bind(this);
+  }
+
+  calchandler(e) {
+    // if (e.target.innerText === 'AC') {
+    //   // this.value = 0;
+    //   // this.num = '';
+    //   const { total, next, operation } = calculate(this.state, e.target.innerText);
+    //   // alert(total);
+    //   this.setState({
+    //     screen: 0, total, next, operation,
+    //   });
+    // } else {
+    //   // this.num += e.target.innerText;
+    //   const { total, next, operation } = calculate(this.state, e.target.innerText);
+    //   alert(total);
+    //   // this.value = total;
+    //   // alert(this.value);
+    //   this.setState({
```

Comment:
- [ ] Please remove any commented-out code through your files for cleaner and more optimized code 😄 🙏